ui/src/components/Menu.js

```diff
@@ -72,21 +72,20 @@ export default function MenuBar() {
                 <core.Divider />
                 {/* LIST 1 */}
                 {/* Leave this alone for now */}
+                {/* Data in this list is read from local file */}
```

Comment:
Is this data still being read from a local file? If not we should change these comments.

Reply:
those are being read from local. the list 1 stands for Home, LeaderBoard, and Challenge Selection